src/main/java/com/melon/portfoliomanager/models/PortfolioItem.java

```diff
@@ -35,6 +35,14 @@ public PortfolioItem(Long userId, String companyName, Double quantity, Double to
         this.totalSoldPrice = totalSoldPrice;
     }
 
+    public PortfolioItem(String companyName, Double quantity, Double totalBoughtPrice, Double totalSoldPrice) {
```

Comment:
If you use this constructor only in tests, then, it is redundant. You should not write code only using it inside tests.